recipes/clize/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "clize" %}
+{% set version = "4.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f54dedcf6fea90a3e75c30cb65e0ab1e832760121f393b8d68edd711dbaf7187
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - attrs >=19.1.0,<20
```

Comment:
Usually `python` and `pip` are the only dependencies that go to `host` - everything else should be in `run`.